Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Dockerfile for wrroc-ga4gh-cloud-converter
+FROM python:3.11.6
+WORKDIR /app
+COPY ./requirements.txt /app/requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
I strongly suggest switching to the `pyproject.toml`/Poetry stack for packaging your project and its dependencies. `setup.py` is deprecated, and `requirements.txt` is also not great.

I would suggest to prepare a PR just for that first, i.e., only add a `pyproject.toml` file with your dependencies. You can check [here](https://github.com/elixir-cloud-aai/TESK/tree/main) how @JaeAeich did it, and you could also ask in the #gsoc channel on Slack if you run into any problems.

Once `pyproject.toml` has been added, you can merge changes into this branch and then update this PR (remove `requirements.txt` and update the calls to install Poetry and install the dependencies in the container).